Zend/zend_compile.h

```diff
@@ -395,6 +401,7 @@ typedef struct _zend_oparray_context {
 
 
 #define ZEND_ACC_PPP_MASK  (ZEND_ACC_PUBLIC | ZEND_ACC_PROTECTED | ZEND_ACC_PRIVATE)
+#define ZEND_ACC_PPP_SET_MASK  (ZEND_ACC_PROTECTED_SET | ZEND_ACC_PRIVATE_SET)
```

Comment:
Can we please name this `ZEND_ACC_PP_SET_MASK`? PPP means public, protected, private, but this only contains two...

Reply:
But you are also quite often using `ZEND_ACC_PPP_SET_MASK | ZEND_ACC_PUBLIC_SET` in compiler, so maybe have both...

Reply:
The public set is comp time only, it's not otherwise used. I'll see if I can improve the naming.

Reply:
Maybe put some define in compile.c which includes public_set? But I think you should remove one `P` from zend_compile.h

Reply:
I added PUBLIC_SET to the mask instead, and in the few cases where we know it's not expected I added an assert instead.